recipes/srahunter/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: srahunter
+  version: 0.0.2
+
+source:
+  url:    https://github.com/GitEnricoNeko/SRAHunter/archive/refs/tags/0.0.2.tar.gz
```

Comment:
It's best to use a jinja template string for the version - so that it only needs to be changed in once place when updating, check other recipes on how to do this. E.g:

```
{% set name = "foo_pkg-123.test" %}
{% set version = "1.2.4" %}
```

and 
```
package:
  name: {{ name|lower }}
  version: {{ version }}
```

and also use that then in URL:

`https://github.com/GitEnricoNeko/SRAHunter/archive/refs/tags/{{version}}.tar.gz`